### ISSUE TITLE
2017-12-14: Rubyfuza

### ISSUE THREAD

**Issue by kennethkalmer** [catehstn/technically-speaking#535]
### Rubyfuza 2018
Africa's Premier Ruby Conference

Event date: 1-2 Feb, 2018
Location: Cape Town, South Africa
Submission deadline: 2017-12-04 08:00 (GMT+2)
Travel expenses covered: No
[http://www.rubyfuza.org/](http://www.rubyfuza.org/)
[CFP - http://cfp.rubyfuza.org/events/rubyfuza-2018-cfp](http://cfp.rubyfuza.org/events/rubyfuza-2018-cfp)
[Code of Conduct - http://www.rubyfuza.org/coc/](http://www.rubyfuza.org/coc/)
[@rubyfuza](https://twitter.com/rubyfuza)

**Comment by chiuki** [comment]
Sorry we don't include conferences that flat out say no to helping speakers. We understand that there may not be budget to support everyone, but we prefer to include conferences that at least make an effort on a case-by-case basis.

**Comment by kennethkalmer** [comment]
Thanks! I'll get some feedback from the rest of the organizers. I'm on the fringes of the effort, helping with social media and awareness side.

**Comment by marcheiligers** [comment]
@chiuki I just want to clarify our position with helping speakers. For context, I am the primary organizer of Rubyfuza. 

It's not that we don't want to help speakers. We have helped speakers in the past by assisting in finding a willing sponsor to help and will continue to do so where possible. 

The fact is, we run on an incredibly small budget. We are a not-for-profit conference and nobody makes any money from it. To give you an idea, a single RailsConf Diamond sponsorship (that's $50k, right?) would pay for the entire conference and we'd be able to make the tickets free.

For the first two years of the conference I bought my own conference ticket, as did every single speaker bar the keynote speaker (there were no other organizers back then, just me). Fortunately we've moved on, speakers and organizers no longer have to purchase their own tickets, but they do still pay for their own travel and accommodation. 

I respect your decision not to list us; I just didn't want to leave this here without further response.
